utilities/is-05-control/is05Control.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+
+# Copyright (C) 2019 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import requests
+import os
+import json
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--ip", required=True, help="IP address or Hostname of DuT")
+parser.add_argument("--port", type=int, required=True, help="Port number of IS-05 API of DuT")
+parser.add_argument("--sender", action="store_true", help="Configure NMOS Sender")
+parser.add_argument("--receiver", action="store_true", help="Configure NMOS Receiver")
+parser.add_argument("-e", "--enable", action="store_true", help="Set master_enable=True")
+parser.add_argument("-d", "--disable", action="store_true", help="Set master_enable=False")
+parser.add_argument("-r", "--request", required=True, help="Patch file to be sent in the request")
+parser.add_argument("-s", "--sdp", help="SDP file to be sent in the request")
+parser.add_argument("-u", "--uuid", required=True, help="UUID of resource to be configured")
+args = parser.parse_args()
+
+# Configure for Sender or Receiver
+if args.sender:
+    print("Configuring NMOS Sender using IS-05")
+    url = "http://{}:{}/x-nmos/connection/v1.0/single/senders/{}/staged".format(args.ip, args.port, args.uuid)
```

Comment:
Another place where I think we now ought to allow ``--version v1.1`` or something on the command line to configure this.